.github/workflows/at_server.yaml

```diff
@@ -711,6 +726,125 @@ jobs:
+          docker tag atsigncompany/secondary:prod-${{ env.VERSION }} atsigncompany/secondary:prod-${{ env.VERSION }}-arm64
+          docker push atsigncompany/secondary:prod-${{ env.VERSION }}-arm64
+      
+      - name: Create and push multi-arch prod manifest
+        run: |
+          docker manifest create atsigncompany/secondary:prod \
+            --amend atsigncompany/secondary:prod-${{ env.VERSION }}-arm64 \
+            --amend atsigncompany/secondary:prod-${{ env.VERSION }}-amd64
+          docker manifest push atsigncompany/secondary:prod
+
+      - name: Create and push multi-arch prod-version manifest
+        run: |
+          docker manifest create atsigncompany/secondary:prod-${{ env.VERSION }} \
+            --amend atsigncompany/secondary:prod-${{ env.VERSION }}-arm64 \
+            --amend atsigncompany/secondary:prod-${{ env.VERSION }}-amd64
+          docker manifest push atsigncompany/secondary:prod-${{ env.VERSION }}          
+  
+  # Rollback canary secondary image.
+  rollback_canary_secondary_image:
+    if: ${{ github.event.inputs.rollback_canary_secondary_image == 'true' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.0.2
+      
+      # Extract version for docker tag
+      - name: Get version
+        run: echo "VERSION=${GITHUB_REF##*/}" >> $GITHUB_ENV
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2.0.0
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Create amd64 labels
+        run: |
+          docker pull --platform=linux/amd64 atsigncompany/secondary:canary-${{ env.VERSION }}
+          docker tag atsigncompany/secondary:canary-${{ env.VERSION }} atsigncompany/secondary:canary-${{ env.VERSION }}-amd64
+          docker push atsigncompany/secondary:canary-${{ env.VERSION }}-amd64
+      
+      - name: Create and push multi-arch canary manifest
+        run: |
+          docker manifest create atsigncompany/secondary:canary \
+            --amend atsigncompany/secondary:canary-${{ env.VERSION }}-amd64
+          docker manifest push atsigncompany/secondary:canary
+
+      - name: Create and push multi-arch canary-version manifest
+        run: |
+          docker manifest create atsigncompany/secondary:canary-${{ env.VERSION }} \
+            --amend atsigncompany/secondary:canary-${{ env.VERSION }}-amd64
+          docker manifest push atsigncompany/secondary:canary-${{ env.VERSION }}  
+
+  # Deploy canary image to secondary image
+  deploy_canary_image_to_prod_secondary_image:
+    if: ${{ github.event.inputs.deploy_canary_image_to_prod_secondary_image == 'true' }}
+    env:
+      working-directory: at_server
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3.0.2
+
+      # Extract version for docker tag
+      - name: Get version
+        run: echo "VERSION=${GITHUB_REF##*/}" >> $GITHUB_ENV
```

Comment:
Which branch would we run this workflow on?

Reply:
When running it we pick a tag for a previous release.